README.md

```diff
@@ -9,55 +9,44 @@ features.
-device. Examples include an Eideticom IOMEM device, a Microsemi NVRAM 
+1. In order to run any P2P traffic, you'll need to have a (Linux based) OS
+that supports [p2pdma][2]. This module is available in all Linux kernels 4.20
+or newer, however it is NOT compiled in by default. You will almost certainly
+have to compile a kernel from source and install that. The instructions on how
+to do this are beyond the scope of this document but there is a tool that can
+help [here][3].
+
+1. Make sure your system has at least one p2pdma capable
+device. Examples include an Eideticom IOMEM device, a Microsemi NVRAM
 card or a CMB enabled NVMe SSD that supports the WDS and RDS features
-(e.g. The Everspin NVNitro card or the Eideticom NoLoad
-HDK). Basically this is any PCI EP capable of exposing a BAR with a
-driver that ties into the p2pmem framework.
+(e.g. The Everspin NVNitro card or the [Eideticom NoLoad<sup>TM</sup> device][4]).
+Basically this is any PCI EP capable of exposing a BAR with a driver that ties
+into the p2pdma framework.
 
-0. In addition to the p2pmem capable device you need at least one
+1. In addition to the p2pmem capable device you need at least one
 other NVMe SSD. This does not have to be CMB enabled (but it is OK if
 it does support CMB), any standard NVMe SSD will do. Perferably you
 will have more than one NVMe SSD but its not a requirement.
 
-0. Ideally place the two (or more) devices noted in the two previous
-points behind a PCIe switch (for example the Microsemi Switchtec or a
-PLX switch). If you do not have a switch you can connect both devices
-to the Root Complex (RC) on the CPU but two things *may* happen:
-  0. Performance may drop. Many RCs are ineffecient at routing P2P
-  traffic.
-  0. It might not work at all. Many RCs block P2P traffic.
-
-0. You almost certainly want to disable the IOMMU in your system via
-both the BIOS and the kernel. Even with these steps some BIOS still
-activate PCIe Access Control Services (ACS) and part of ACS includes
-TLP redirection which kills p2pmem. If this is happening on your setup
-you can try and disable ACS or use this patch [2] to get the kernel to
-turn it off. Use with care as it does bad things to IOVA sharing.
-
-0. Boot your system. Assuming your p2pmem driver (which might be the
-NVMe driver in your case) bound to the PCI EP exposing the p2pmem you
-should see some lines in dmesg regarding p2pmem. You should also see a
-device for each p2pmem in your system (e.g. /dev/p2pmemN where N is a
-numnber). If you don't check all the steps above. Note there is also a
-debugsfs entry for each p2pmem device in <debugfs mount
-point>/p2pmem/p2pmemN.
-
-0. Compile p2pmem-test by runnning make. Note p2pmem-test using
-libargconfig as a submodule so make sure you have run submodule init
-to pull that code.
-
-0. You can now run p2pmem-test. Use p2pmem-test -h to get a list of
-options.
+	Ideally place the two (or more) devices noted in the two previous points behind
```

Comment:
Is this indent intended?

Reply:
Yes. This is so that it shows up as part of the number above it.

Reply:
I've updated it to be a part of the previous block.